src/core/messages.ts

```diff
@@ -578,12 +556,35 @@ export class DefaultMessages
     return deletedMessage;
   }
 
+  react(message: Message, reaction: string): Promise<void> {
+    this._logger.trace('Messages.react();', { message, reaction });
+    return this._chatApi.reactToMessage(this._roomId, message.serial, reaction);
+  }
+
   /**
    * @inheritdoc Messages
    */
-  subscribe(listener: MessageListener): MessageSubscriptionResponse {
-    this._logger.trace('Messages.subscribe();');
-    super.on([MessageEvents.Created, MessageEvents.Updated, MessageEvents.Deleted], listener);
+  subscribe(listener: MessageListener | MessageListenerObject): MessageSubscriptionResponse {
```

Comment:
Given we're in the 0.x versions, I'd be comfortable with just making the breaking change - rather than having both versions

Reply:
Happy to remove the `MessageListener` part